hugegraph-dist/release-docs/NOTICE

```diff
@@ -922,3 +922,14 @@ SPDX-License-Identifier: BSD-3-Clause
 The project maintains the following source code repositories:
 
 * https://github.com/eclipse-ee4j/jaf
+========================================================================
+
+okhttp NOTICE
+
+========================================================================
+
+Note that publicsuffixes.gz is compiled from The Public Suffix List:
+https://publicsuffix.org/list/public_suffix_list.dat
+
+It is subject to the terms of the Mozilla Public License, v. 2.0:
+https://mozilla.org/MPL/2.0/
```

Comment:
be careful about the `MPL2.0`, it's under Category B in ASF (binary usage is allowed, just a reminder)